bokehjs/src/coffee/models/ranges/data_range1d.coffee

```diff
@@ -22,7 +22,7 @@ export class DataRange1d extends DataRange
   }
 
   @internal {
-      mapper_hint:     [ p.String, 'auto' ]
+      scale_hint:     [ p.String, 'auto' ]
```

Comment:
yes this PR is already big and disruptive, and that would be an isolated chunk of work once this PR is in